code/game/machinery/machinery.dm

```diff
@@ -246,6 +246,12 @@ Class Procs:
 		return
 	if(!isturf(M.loc) && M.loc != src)
 		return
+	if(getBrainLoss() >= 60)

```

Comment:
srsly
I'll see if I can find a non-horribly ugly way to work around this cos I don't really want to deal with fixing all the uses of canUseTopic where they shouldn't be.